src/Chainweb/Version.hs

```diff
@@ -927,6 +927,19 @@ chainweb213Pact Development = (>= 95)
 chainweb213Pact (FastTimedCPM g) | g == petersonChainGraph = (> 25)
 chainweb213Pact _ = const True
 
+-- | Omnibus pact and coin contract changes for 2.14
+--
+-- - coin v4 upgrade
```

Comment:
LOL we totally didn't fill out the rest of the changes in the comment on 2.13, so lets skip the laundry list here and in that comment (L920-921)

Reply:
Fixed